### ISSUE TITLE
Minor correction to WIKI (including How-to-get-it WIKI page )

### ISSUE THREAD

**Issue by mindrail** [lgblgblgb/xemu#344]
On page: https://github.com/lgblgblgb/xemu/wiki/MEGA65-ROM-%22how-to-get-it%22-tutorial-for-Xemu

At the bottom of the page:

`Right click into the running emulator window, select the "SD-card" menu, and the "Update files on the SD image" sub-menu. You will be prompted for the ROM, so you need to select it. Xemu will update the SD-card image for you. At this point, if you encounter any problem after the update, you can try to exit Xemu and re-start, though in newer Xemu versions this shouldn't be an issue.`

`Right click into the running emulator window, select the menu item "Disks" > "SD-card" > "Update files on SD image".`

Just a minor correction to the correct menu path and perhaps easier wording. 

***

2nd Minor correction:

On page: https://github.com/lgblgblgb/xemu/wiki/VIC-IV

the link: https://github.com/MEGA65/mega65-core/blob/master/doc/viciv-modes.md

should be https://github.com/MEGA65/mega65-core/blob/master/docs/viciv-modes.md

:) Thanks for your amazing work!!

**Comment by lgblgblgb** [comment]
Yes indeed, thanks! This is because it was written when the menu was different. So actually it is correct just outdated ;) I'm afraid there is lots of outdated info everywhere, I should take my time (just need to find that time ...) to read my blah-blah pages everywhere and correct outdated information.

**Comment by lgblgblgb** [comment]
f4df488913f47b2203e4ad38f3b737e51dd4b53f

**Comment by mindrail** [comment]
That's all good, I know it can be hard finding the time! Happy to have a look through some more of the wiki if I get a chance.

Dont forget the "2nd minor change" for the VIC IV link further down in that issue :)

**Comment by lgblgblgb** [comment]
OMG. Thanks for mentioning, I was not aware at all that you have more things in this issue ... I really think, that I should delete these pages. Totally pointless, especially if you see the date, about 6 years old (from 2016) ... I guess it was made when I was writing the very first versions of MEGA65 emulation and I used wiki as a kind of "notebook" for mainly myself ...

**Comment by lgblgblgb** [comment]
Some obsoleted pages has been removed including VIC-IV.
